src/components/text/Text.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import clsx from 'clsx';
+
+import Type, { TTypeWeight } from '../../foundations/type';
+
+import { PText } from './Text.props';
+import {
+    DEFAULT_TEXT_COLOR,
+    DEFAULT_TEXT_MARGIN,
+    DEFAULT_TEXT_SIZE,
+    DEFAULT_TEXT_ELEMENT,
+    DEFAULT_TEXT_WEIGHT,
+} from './Text.constants';
+
+const Text: React.FC<PText> = ({
+    children,
+    className,
+    size = DEFAULT_TEXT_SIZE,
+    element = DEFAULT_TEXT_ELEMENT,
+    weight = DEFAULT_TEXT_WEIGHT,
+    margin = DEFAULT_TEXT_MARGIN,
+    color = DEFAULT_TEXT_COLOR,
+}): JSX.Element => (
+    <Type
+        className={clsx(className, 'Text')}
+        element={element}
+        size={size}
+        margin={margin}
+        color={color}
+        weight={weight as TTypeWeight}
+        role={'text'}
```

Comment:
I was initially going to suggest this isn't needed here either, however, it turns out that screen readers may read native text elements awkwardly when there are child `<span>`'s (for instance) that surround portions of the text and this can help them to read everything as text in one go. Given we allow for user-defined children, might be good to keep.

We should test this though at some point.

Reply:
Interesting, despite what I read, according to the AXE storybook plugin, `role=text` is not a valid role. 🤷‍♂️

Reply:
i will just leave it in there for now ... we will see if it has any effect